src/app/career-table/career-table.component.ts

```diff
@@ -3,18 +3,19 @@ import { DataService } from "../_services/data.service";
 import { NgbModal } from "@ng-bootstrap/ng-bootstrap";
 import { RoleInformationComponent } from "../role-information/role-information.component";
 import { Role } from "../_classes/role";
+import { TestBed } from '@angular/core/testing';
```

Comment:
This import isn't used in this file. It's also meant to be used for writing automated tests.